charts/camunda-platform/values.yaml

```diff
@@ -1746,6 +1746,105 @@ web-modeler:
       # Postgresql.auth.database defines the name of the database to be created for Web Modeler
       database: web-modeler
 
+# Connectors configuration for the Connectors.
+connectors:
+  # Enabled if true, the Connectors deployment and its related resources are deployed via a helm release
+  enabled: true
+
+  # Switch for inbound mode (e.g., for webhook or polling)
+  inbound:
+    # Acceptable values: disabled, credentials, or oauth
+    mode: disabled
+
+  # Image configuration to configure the Connectors image specifics
+  image:
+    # Image.registry can be used to set container image registry.
+    registry: ""
+    # Image.repository defines which image repository to use
+    repository: camunda/connectors-bundle
+    # Image.tag can be set to overwrite the global tag, which should be used in that chart
+    tag: 0.16.1
+    # Image.pullSecrets can be used to configure image pull secrets https://kubernetes.io/docs/concepts/containers/images/#specifying-imagepullsecrets-on-a-pod
+    pullSecrets: []
+
+  # PodAnnotations can be used to define extra Connectors pod annotations
+  podAnnotations: {}
+  # PodLabels can be used to define extra Connectors pod labels
+  podLabels: {}
+
+  # Logging configuration for the Operate logging. This template will be directly included in the Operate configuration yaml file
```

Comment:
this is for operate, does the connectors deployment need it?

Reply:
Good catch. It is `io.camunda.connector: DEBUG` in properties. Everything as expected, just missed to adjust comment. Done in the next revision.